test/parallel/test-timers-ordering.js

```diff
@@ -1,22 +1,22 @@
 'use strict';
 require('../common');
-var assert = require('assert');
-var Timer = process.binding('timer_wrap').Timer;
+const assert = require('assert');
+const Timer = process.binding('timer_wrap').Timer;
 
-var N = 30;
+const N = 30;
 
-var last_i = 0;
-var last_ts = 0;
+let last_i = 0;
+let last_ts = 0;
 
-var f = function(i) {
+const f = function(i) {
   if (i <= N) {
     // check order
     assert.equal(i, last_i + 1, 'order is broken: ' + i + ' != ' +
```

Comment:
Since you're already in here refactoring, could you also change the `assert.equal` on this line to `assert.strictEqual()`?